src/robot/libraries/BuiltIn.py

```diff
@@ -613,10 +613,21 @@ def should_be_equal(self, first, second, msg=None, values=True):
         for example, string ``false`` or ``no values``. See `Boolean arguments`
         section for more details.
 
+        If ignore_case is True, it indicates that ``first`` and ``second`` should be
+        compared case-insensitively.  See `Boolean  arguments` section for more details.
+        (This option is new in Robot Framework 3.0.1)
```

Comment:
`This option is new in Robot Framework 3.0.1.` without parenthesis would be more consistent with other similar notes.